scripts/_helpers.py

```diff
@@ -60,7 +60,8 @@ def check_config_version(config, fp_config=CONFIG_DEFAULT_PATH):
             f"The current version of 'config.yaml' doesn't match to the code version:\n\r"
             f" {current_config_version} provided, {actual_config_version} expected.\n\r"
             f"That can lead to weird errors during execution of the workflow.\n\r"
-            f"Please update 'config.yaml' according to 'config.default.yaml' ."
+            f"Please update 'config.yaml' according to 'config.default.yaml'\n\r"
+            "and update the environment to the latest version."
```

Comment:
I wonder if it could make sense to recommend also keep a reserve copy of the previous environment (if it's still functional)? The may be also cases when the configuration changes do not require to re-build an environment, while the environments can be quite fragile unfortunately. What do you think?

Reply:
Slightly revised the text

Reply:
Looks good! 🙂